app/src/main/java/com/codesoom/assignment/common/exception/ProductNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package com.codesoom.assignment.common.exception;
+
+/**
+ * 상품이 존재하지 않을 때 예외를 정의한다
```

Comment:
```suggestion
 * 상품이 존재하지 않을 때 던집니다.
```
다음은 `IllegalArgumentException` 클래스의 주석입니다.

![image](https://user-images.githubusercontent.com/1855714/163674933-8e92133c-e63f-414c-b088-9776fc8426c1.png)